### PR TITLE
Modmail Bot Example

### PR DESCRIPTION
**Pull request by richard1805-DE**

### CODE REVIEW

File: modmail/cogs/mmsyt.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+                        if member:
+                            embed = discord.Embed(description= message.content, color = discord.Color.dark_green())     
+                            embed.set_author(name = message.author)
+                            await member.send(embed=embed)
+        except Exception as e:
+            print(e)
+
+    @commands.command()
+    async def close(self, ctx):
+        if ctx.channel.category and ctx.channel.category.name == "Modmail Tickets": # Change to our Modmail tickets category
```
Comment (SimplexDE) [inline modmail/cogs/mmsyt.py:51]: Work with an ID here maybe

File: modmail/cogs/mmsyt.py [hunk 2]
```diff
@@ -0,0 +1,57 @@
+import discord
+from discord import utils
+from discord.ext import commands
+import asyncio
+
+class MmSyt(commands.Cog):
```
Comment (SimplexDE) [inline modmail/cogs/mmsyt.py:6]: change the class name to ModmailSystem for better readablity, also rename the file to modmailsystem

remember to change the setup function according to the name change

File: modmail/cogs/mmsyt.py [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+                embed = discord.Embed(description = message.content, color = discord.Color.dark_green())
+                embed.set_author(name = message.author)
+                await channel.send(embed=embed)
+
+            elif isinstance(message.channel, discord.TextChannel):
+                if message.content.startswith(self.bot.command_prefix):
+                    pass
+                else:
+                    topic = message.channel.topic
```
Comment (SimplexDE) [inline modmail/cogs/mmsyt.py:39]: This includes every channel with a topic?

**Review by SimplexDE — CHANGES_REQUESTED** [review]
Seems to inlcude every channel with a topic, which will cause problems

File: modmail/cogs/mmsyt.py [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+from discord.ext import commands
+import asyncio
+
+class MmSyt(commands.Cog):
+    def __init__(self, bot):
+        self.bot = bot
+
+    @commands.Cog.listener()
+    async def on_message(self, message):
+        try:
```
Comment (Ju0x) [inline modmail/cogs/mmsyt.py:12]: Use on_error instead of big try/except block

File: modmail/cogs/mmsyt.py (hunk 2 above)
Comment (Ju0x) [inline modmail/cogs/mmsyt.py:6]: Better name for the class

File: modmail/cogs/mmsyt.py [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+            if message.author.bot:
+                return
+            
+            if isinstance(message.channel, discord.DMChannel):
+                guild = self.bot.get_guild(12321231123) # Change the guild id to your guild id
```
Comment (Ju0x) [inline modmail/cogs/mmsyt.py:17]: Use variables at the beginning of the file to find these changeable values better or even a settings.json

**Review by Ju0x — DISMISSED** [review]

**Review by SimplexDE — APPROVED** [review]
👍

**Pull request merged**